src/components/Editor/DrawingBoard/Canvas/scheduler.ts

```diff
@@ -0,0 +1,45 @@
+import { Point } from 'features/docSlices';
+
+export type Task = Point;
+
+const INTERVAL_TIME = 50;
+let tasks: Array<Task> = [];
+let timeout: ReturnType<typeof setTimeout>;
+let work: Function | undefined;
+
+const doWork = () => {
+  if (typeof work === 'function') {
+    work(tasks);
+    tasks = [];
+  }
+};
+
+function requestHostCallback(isDone: Boolean) {
```

Comment:
I'm curious why the name is `requestHostCallback ` 🤔

Reply:
I used the name by executing the previously registered callback, but now the callback disappears, The name is weird as you said. I will change it :)